common/src/main/java/io/netty/util/HashedWheelTimer.java

```diff
@@ -279,6 +290,112 @@ public HashedWheelTimer(
         }
     }
 
+    /**
+     * Creates a new timer with the default thread factory
+     * ({@link Executors#defaultThreadFactory()}) and default number of ticks
+     * per wheel.
+     *
+     * @param tickDurationMillis the duration between tick
```

Comment:
Maybe say explicitly it's in milliseconds? s/the duration between tick/the duration between ticks in milliseconds/